packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -73,10 +78,19 @@ export const getCFDAccountKey = ({ market_type, sub_account_type, platform, shor
                 case 'vanuatu':
                     return 'financial_v';
                 default:
-                    return 'financial';
+                    return 'financial_demo';
             }
         }
     }
+    if (market_type === 'all' && platform === CFD_PLATFORMS.MT5) {
```

Comment:
Add this as an else-if condition to the condition above for the rest of the `all` market_type

Reply:
Done